tests/test_permission_to_create_is_supported.py

```
@@ -0,0 +1,138 @@
+            "birth_date": "1990-01-01",
+            "books": [
+                book.id  # Referencing related resource created by the user themselves
+            ],
+        },
+    )
+    assert response.status_code == status.HTTP_201_CREATED, response.json()
+    a = models.Author.objects.get(id=response.json()["id"])
+    assert a.name == "Some author"
+    assert a.birth_date == datetime.date(1990, 1, 1)
+    assert a.books.count() == 1
+    assert (b := a.books.first()) is not None and b.id == book.id
```

Ruff (0.8.2)

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)